.github/workflows/tag.yml

```diff
@@ -68,7 +68,11 @@ jobs:
         with:
           ref: ${{ inputs.ref }}
           token: ${{ secrets.BROADBOT_TOKEN }} # this allows the push to succeed later
-          persist-credentials: false
+      - name: Setup Git
+        shell: bash
+        run: |
+          git config --global user.name 'broadbot'
+          git config --global user.email 'broadbot@broadinstitute.org'
```

Comment:
```suggestion
```
I don't think this step is needed, I think just removing persist-credentials will be sufficient. We have other GHA examples that don't need this configurations (like sam  and terra-ui: https://github.com/DataBiosphere/terra-ui/blob/6387e1a2b1947840e1cfea3c6b78c5a2f48e9c35/.github/workflows/tag-publish.yaml#L47)

Reply:
Sure, makes sense, let's try it this way and if we need this I can open a separate PR.